pack/rpm.mk

```diff
@@ -101,7 +132,8 @@ $(BUILDDIR)/$(RPMSPEC): $(RPMSPECIN)
 #
 $(BUILDDIR)/$(RPMSRC): $(BUILDDIR)/$(TARBALL) \
                        $(BUILDDIR)/$(RPMSPEC) \
-                       prebuild-$(OS)-$(DIST)
+                       prebuild-$(OS)-$(DIST) \
+                       prebuild_cleanup
```

Comment:
I guess it should be before a user-provided prebuild script.

Reply:
Corrected.